doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -678,26 +650,25 @@ or focus the next one.
    indentation. For example ``-`` must be the outer bullet and ``**`` the inner
    one in the example below.
```

Comment:
The note does not match anymore the modified example.

Reply:
Got it.

FWIW, your comment reads oddly (ungrammatically?) because "anymore" is out of place.  It should be "The note does not match the modified example **anymore**."  I see similar mistakes by others.  Does French expect adverbs to be next to the verb they modify?

Reply:
Yes, I think that's very likely a Frenchism. In this case, the French version would be "ne correspond plus" (standard negative construction).